packages/engineer/src/cli-commands.ts

```diff
@@ -131,8 +131,8 @@ export function buildCommand(program: typeof commander) {
         .option('--external [true|false]', 'build feature as external', parseBoolean, false)
         .option('--eagerEntrypoints [true|false]', 'build feature as external', parseBoolean, false)
         .option(
-            '--featureOutDir <featureOutDir>',
-            'the directory where the published feature file is located (relative to the base path). default: "."'
+            '--libOutDir --featureOutDir <featureOutDir>',
```

Comment:
reverted the lib outDir
this means the "lib/dist" folder, to where all the sources that will be consumed, will be located at.
for typescript projects this will usually be the outDir